apps/bfd-pipeline/bfd-pipeline-idr/model.py

```diff
@@ -616,10 +622,26 @@ def _current_fetch_query(start_time: datetime) -> str:  # noqa: ARG004
 
 
 def claim_type_clause(start_time: datetime) -> str:
+    fetch_latest_claims = "IDR_LATEST_CLAIMS" in os.environ and os.environ["IDR_LATEST_CLAIMS"] in (
+        "1",
+        "true",
+    )
+    add_latest_claim_ind = (
+        f""" AND {ALIAS_CLM}.clm_ltst_clm_ind = 'Y' """ if fetch_latest_claims else ""
```

Comment:
Can use single quotes here since it's not a multi-line string

```suggestion
        f" AND {ALIAS_CLM}.clm_ltst_clm_ind = 'Y' " if fetch_latest_claims else ""
```